scripts/apps/authoring/metadata/metadata.ts

```diff
@@ -357,30 +357,31 @@ function MetaDropdownDirective($filter) {
         },
         templateUrl: 'scripts/apps/authoring/metadata/views/metadata-dropdown.html',
         link: function(scope, elem) {
-            scope.listFields = ['place', 'genre', 'anpa_category', 'subject', 'authors'];
+            scope.multiInputFields = ['place', 'genre', 'anpa_category', 'subject', 'authors'];
 
             scope.select = function(item) {
-                var o = {};
+                var fieldObject = {};
```

Comment:
What does the object contain/is used for? Could you add types to it?

Reply:
It contains data like this
![custom-cv](https://user-images.githubusercontent.com/10600741/56353223-8b60d300-61ee-11e9-8f79-da58aea86ff8.png).

Reply:
You can type it as `fieldObject: {[fieldId: string]: any}` since it's hard to type it more specifically due to the structure.